data/moves.ts

```diff
@@ -19284,12 +19284,14 @@ export const Moves: import('../sim/dex-moves').MoveDataTable = {
 		pp: 10,
 		priority: 0,
 		flags: {snatch: 1, heal: 1, metronome: 1},
-		onTry(source) {
+		onTry(source, target, move) {
+			if (move.sourceEffect === 'snatch' && this.gen >= 5) return;
```

Comment:
i think it might be better to use the gen5 mod files for this since thats the system we have in place currently

Reply:
fair enough; honestly someday i wonder if we should adopt using this.gen in event callbacks more, but i won't unilaterally go against the status quo